crates/hulk_behavior_simulator/src/state.rs

```diff
@@ -205,17 +204,8 @@ impl State {
                 } else {
                     None
                 };
-            robot.database.main_outputs.primary_state =
-                match (robot.is_penalized, self.game_controller_state.game_state) {
-                    (true, _) => PrimaryState::Penalized,
-                    (false, GameState::Initial) => PrimaryState::Initial,
-                    (false, GameState::Standby) => PrimaryState::Standby,
-                    (false, GameState::Ready { .. }) => PrimaryState::Ready,
-                    (false, GameState::Set) => PrimaryState::Set,
-                    (false, GameState::Playing { .. }) => PrimaryState::Playing,
-                    (false, GameState::Finished) => PrimaryState::Finished,
-                };
```

Comment:
Why removed?

Reply:
This was a simulator internal fake for the `game_controller_state_filter'. We now properly run this filter as a full node so this fake is no longer needed.